web/stories/ui/formatted-text.stories.svelte

```diff
@@ -0,0 +1,227 @@
+<script context="module" lang="ts">
+  import imgExample2 from '../../public/stories-assets/images/long_width_example.avif'
```

Comment:
If we are adding new images, it is better to put them to `web/stories/ui/assets/`.

The best way to get images is Wikipedia since all of they are free (EM logo is not open source).

Reply:
Also, we need a way to avoid adding those images to Size Limit. For instance, we can use some prefix and add its ignore to `web/.size-limit.json`. Or we can configure Vite to put all `web/stories/` images to some subdir (and ignore it in the same `web/.size-limit.json`).

Reply:
I have thoughts about this case. Storybook builds independently, but assets from it now are copied to the `dist/assets` folder of our main project. However, if we place these assets, for example, in `web/stories/assets`, it will go to `dist/ui/assets`, to the stories folder. In this case, size-limit will skip the check (since it already ignores `!dist/ui/**`), and we'll divide the files of packages. I don't think we'll use these images in the main project ? What do you think ?

Reply:
Nice idea